src/server.ts

```diff
@@ -375,18 +375,17 @@ const createParticipant = async (tag: string): Promise<ParticipantInfo> => {
 const deleteSession = async () => {
   if (sessionId) {
     try {
-      await webRTCController.deleteSession(accountId, sessionId);
+      await webRTCController.deleteSession(accountId, "garbage");
```

Comment:
is there a reason we put "garbage" in here?

Reply:
my bad - that was for testing and I screwed up the commit/push.
I fixed it with the last commit.